app/src/components/explorer/yasgui.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div :id="id"></div>
+</template>
+
+<script>
+import YASGUI from '@triply/yasgui'
+
+export default {
+  name: 'yasgui',
+  props: {
+    id: {
+      type: String,
+      default: () => 'YASGUI'
+    }
+  },
+  mounted () {
```

Comment:
NIT: Should this be triggered on the mounted lifecycle

Reply:
I was imitating our yasqe and yasr implementations here, which both used mounted. I think it was originally done this way because they needed to be able to access/modify `this.$el` ?